css/main.css

```diff
@@ -18,7 +18,7 @@ hr {
   width: 80vw;
 }
 
-header {
+#header {
   display: flex;
   justify-content: space-between;
   align-items: center;
```

Comment:
- [ ] As the previous code reviewer pointed out. Please make sure the logo disappears when the user clicks on the hamburger button and also the hamburger button must change to (X).

Your design 👇

![re-review](https://user-images.githubusercontent.com/66992901/190683549-3997e607-a57e-4e57-bb23-f001113789c6.PNG)

Figma design 👇

![figma](https://user-images.githubusercontent.com/66992901/190683704-fa2824f0-3ccd-4e39-8ec4-f77026860cef.PNG)